README.md

```diff
@@ -91,6 +91,8 @@ Available options:
         connection rate will take precedence if both are set.
         NOTE: if using rate limiting and a very large rate is entered which cannot be met,
               Autocannon will do as many requests as possible per second.
+  -C/--correctCoordinatedOmission
```

Comment:
Can you add some docs to explain why this is needed? Also, it seems something we might want to enable by default? Why somebody would not want this?

Reply:
Well you are right it could be enable by default, like in wrk2. The downside would be for users that do benchmarks on a regular basis and track the performance trends. Do not know if this would impact many users though...  
If It is enable by default would you like to be able to deactivate it from the command line or would you like to get in the output corrected and uncorrected results (like in wrk2)?

Reply:
I would recommend to deactivate from the command line. I would also add some reference on the problem as a lot of people would be unfamiliar with it.